src/main/java/com/foundation/salesforce/core/api/OpportunityApi.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * @(#) OpportunityApi.java Copyright (c) 2019 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package com.foundation.salesforce.core.api;
+
+import com.foundation.salesforce.core.restClient.RestClientApi;
+import io.restassured.response.Response;
+
+import java.util.Map;
+
+import static com.foundation.salesforce.core.utils.EndPoints.OPPORTUNITY_ENDPOINT;
+
+/**
+ * OpportunityApi class
```

Comment:
Please add a dot at the end of the prayer.

Reply:
Fixed.